src/app/page/admin/categories/admin.category.service.ts

```diff
@@ -24,6 +26,26 @@ export class AdminCategoryService {
       .catch(this.handleError);
   }
 
+  public createCategory(newCategory: CreateCategoryDTO) {
+    console.log(newCategory);
```

Comment:
I think, an effort was made to remove it.

Reply:
I'll fix it. Thanks